### PR TITLE
✨💥😎 Add banner and webhooks libraries

### LINKED ISSUE #3: Add Banner API
The microservice should interface with the scraper, the scraper should not be built into the microservice itself. This allows for separate scaling of application servers w/ business logic and scrapers.

### PR DESCRIPTION
**Pull request by bmiddha**
- Added webhooks library
- Added banner library
- Added docs
- Updated docker configs
- Updated github actions
- Updated pre-commit hooks

### CODE REVIEW

**Review by alexchomiak — COMMENTED** [review]
How are we going to communicate with the Banner API from within the Microservice??

**Comment by bmiddha** [comment]
The `lib/Banner.ts` library provides Banner	API interaction between this service and Banner.

**Review by alexchomiak — APPROVED** [review]
Everything else looks good! 👍 This is the glue that will bring the rest of our project together. Good work Bharat! Here's two tacos!!! 🌮 🌮

**Comment by bmiddha** [comment]
TWO TACOS!!!! 😮 
tacos

**Comment by alexchomiak** [comment]
It's TACO TIME!!! 🌮 🌮 🌮 🌮 
taco

File: scripts/compose-install-all.ps1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$services = "api","banner","client"
+docker-compose up -d
+
+ForEach($service in $services) {
+    docker-compose exec $service npm install
+    docker-compose restart $service
+}
```
Comment (anarmawala) [inline scripts/compose-install-all.ps1:7]: This is awesome!

**Review by anarmawala — APPROVED** [review]
Look's good! Here is a 🌮 for you!

**Pull request merged**